cms/djangoapps/contentstore/views/public.py

```diff
@@ -34,7 +33,6 @@ def signup(request):
     return render_to_response('register.html', {'csrf': csrf_token})
 
 
-@ssl_login_shortcut
```

Comment:
@rlucioni Sorry about that, actually I really don't know what we have to remove and what we don't from external_auth. Now I just tried to remove ```_signup()```. Have another look please.